docker/wasi-sdk.cmake

```diff
@@ -18,6 +18,7 @@ set(CMAKE_AR /usr/bin/llvm-ar-$ENV{LLVM_VERSION})
 set(CMAKE_RANLIB /usr/bin/llvm-ranlib-$ENV{LLVM_VERSION})
 set(CMAKE_C_COMPILER_TARGET ${triple})
 set(CMAKE_CXX_COMPILER_TARGET ${triple})
+set(CMAKE_ASM_COMPILER_TARGET ${triple})
```

Comment:
When cribbing from top-level cmake file, I noticed this line was there that hadn't been added here